analyzer-alert/src/main/java/cloud/erda/analyzer/alert/functions/AlertEventTemplateRenderFunction.java

```diff
@@ -82,16 +82,16 @@ public RenderedAlertEvent map(AlertEvent value) throws Exception {
 
         templateContext = processFormatter(templateContext, value.getAlertNotifyTemplate().getFormats());
 
-        TemplateRenderer titleRenderer = templateManager.getRenderer(value.getAlertNotifyTemplate().getId() + "_title", value.getAlertNotifyTemplate().getTitle(), value.getAlertNotifyTemplate().isVariable());
+        TemplateRenderer titleRenderer = templateManager.getRenderer(value.getAlertNotifyTemplate().getAlertIndex() + "_title", value.getAlertNotifyTemplate().getTitle(), value.getAlertNotifyTemplate().isVariable());
```

Comment:
Why use alertindex as the cache key, which will cause exceptions in different notification methods templates